app/models/registration.rb

```diff
@@ -398,11 +398,8 @@ def permit_user_cancellation?
   end
 
   def consider_auto_close
-    if outstanding_entry_fees == 0
-      return competition.attempt_auto_close!
-    else
-      return false
-    end
+    return competition.attempt_auto_close! if outstanding_entry_fees == 0
+    false
```

Comment:
```suggestion
    outstanding_entry_fees.zero? && competition.attempt_auto_close!
```

Reply:
Whenever you have `if`s flying around, and you manually return booleans at the end, then there's a ~75% chance that your expression can be condensed into the `if`s.
Manually returning "naked" booleans is aaaalmost never necessary.

Reply:
Thanks!